modules/vba_documents/lib/vba_documents/multipart_parser.rb

```diff
@@ -139,7 +159,7 @@ def self.consume_body_tempfile(lines, separator)
                                               detail: 'Unexpected end of payload')
         end
         linechomp = line.chomp(LINE_BREAK)
-        if linechomp == "#{separator}--"
+        if (linechomp == "#{separator}--") || (linechomp == "#{separator}--#{CARRIAGE_RETURN}")
```

Comment:
I don't think these parenthesis are needed.

Reply:
Linter wants it